chapters/tutorials/index.rst

```diff
@@ -83,31 +86,35 @@ To add elements to your Flow you just need to use the `add` keyword. You can add
 
     from jina import Flow
 
-    f = Flow().add().add().add()
+    flow = Flow().add().add()
 
-And for our example, we need to add two elements:
+And for our example, we need to add two `Executors`:
 
 1. A transformer (to encode our data)
 2. An indexer
 
+So add the following to our code:
+
 .. code-block:: python
 
     from jina import Flow
-    f = (
+    flow = (
             Flow()
             .add(uses=MyTransformer)
             .add(uses=MyIndexer)
         )
 
-Right now we haven't defined `MyTransformer` or `MyIndexer`, let's create some dummy `Executors` so we can try our code. These will not be our final `Executors` but let's create something basic to learn first.
+Right now we haven't defined `MyTransformer` or `MyIndexer`, let's create some dummy `Executors` so we can try our app. These will not be our final `Executors` but let's create something basic to learn first.
```

Comment:
Right now we haven't defined `MyTransformer` or `MyIndexer`. Let's create some dummy `Executors` so we can try our app. These will not be our final `Executors` but just something basic to learn first.

(punctuation, slight wording)